backend/src/providers/email/smtp.provider.ts

```
@@ -107,10 +107,16 @@ export class SmtpEmailProvider implements EmailProvider {
     variables?: Record<string, string>
   ): Promise<void> {
     const config = await this.getRequiredConfig();
-    const emailTemplate = await EmailTemplateService.getInstance().getTemplate(template);
+    const emailTemplate = await EmailTemplateService.getInstance().getTemplate(template, 'custom_smtp');
```

ESLint

[error] Replace `template,·'custom_smtp'` with `⏎······template,⏎······'custom_smtp'⏎····`

(prettier/prettier)